docs/guide/export.rst

```diff
@@ -127,6 +127,25 @@ TODO: contributors help is welcomed!
 Probably a good starting point: https://github.com/elliotwaite/pytorch-to-javascript-with-onnx-js
 
 
+Export to ONNX / TF / TFLite / Edge TPU
+--------------------------------
+
```

Comment:
I added a section on motivations. Of note, I still left TFLite in the title, because 
a) That's a primary goal for passer-bys, likely moreso than the Coral itself is
b) At least for what I'm currently doing, TFLite turns out to be better fit than the Coral along some dimensions.